src/Sylius/Bundle/ShopBundle/Resources/views/Cart/Summary/_item.html.twig

```diff
@@ -40,12 +40,11 @@
     </td>
     <td class="right aligned">
         {# Todo: Fix this mess... #}
-        {% set regular_total = item|sylius_regular_price %}
         {% if 0 != item.getAdjustmentsTotalRecursively('order_unit_promotion') %}
-            <span class="sylius-total" style="text-decoration: line-through">{{ regular_total|sylius_price }}</span>
+            <span class="sylius-total" style="text-decoration: line-through">{{ item.regularTotal|sylius_price }}</span>
             <span class="sylius-discounted-total">{{ item.total|sylius_price }}</span>
         {% else %}
-            <span class="sylius-total">{{ regular_total|sylius_price }}</span>
+            <span class="sylius-total">{{ item.regularTotal|sylius_price }}</span>
```

Comment:
As you can see above `{# Todo: Fix this mess... #}` that piece of code is wrong. We should display `item.unitPrice` and `item.discountedUnitPrice` if they are different (see order details in admin).

This change will require few adjustments in current promotions scenarios. Maybe we could extract it as a separate PR?